src/model/WindowPropertyFrameAndDivider.hpp

```diff
@@ -75,7 +75,9 @@ namespace model {
 
     bool isFrameInsideProjectionDefaulted() const;
 
-    boost::optional<double> frameConductance() const;
+    double frameConductance() const;
+
+    bool isFrameConductanceDefaulted() const;
```

Comment:
API break on return type for frameConductance() + add a isFrameConductanceDefaulted()